sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/pipeline/TelemetryPipelineResponse.java

```diff
@@ -58,17 +58,11 @@ public boolean isInvalidInstrumentationKey() {
     }
 
     static Set<ResponseError> parseErrors(String body) {
-        JsonNode jsonNode;
-        try {
-            jsonNode = new ObjectMapper().readTree(body);
-        } catch (JsonProcessingException e) {
-            throw new IllegalStateException("Failed to parse response body", e);
+        try (JsonReader reader = JsonProviders.createReader(body)) {
+            Response response = Response.fromJson(reader);
+            return new HashSet<>(response.getErrors());
+        } catch (Exception e) {
+            return emptySet();
```

Comment:
can you add a comment why it's ok to ignore this exception?

Reply:
it is for DiagnosticTelemetryPipelineListener

> if (!errors.isEmpty()) {
                    operationLogger.recordFailure(
                        "Received response code " + responseCode + " (" + String.join(", ", errors) + ")",
                        INGESTION_ERROR);
                }

how do i put that as comment? maybe just saying logging response code?